File: CS/PivotGridCustomization.Module/Updater.cs
using System;

using DevExpress.ExpressApp.Updating;
using DevExpress.Xpo;
using DevExpress.Data.Filtering;
using DevExpress.Persistent.BaseImpl;

namespace PivotGridCustomization.Module {
    public class Updater : ModuleUpdater {
        public Updater(Session session, Version currentDBVersion) : base(session, currentDBVersion) { }
        public override void UpdateDatabaseAfterUpdateSchema() {
            base.UpdateDatabaseAfterUpdateSchema();
            TestObject testObject = Session.FindObject<TestObject>(CriteriaOperator.Parse("Name == 'First'"));
            if (testObject == null) {
                TestObject first = new TestObject(Session);
                first.Name = "First";
                first.Value = 5;
                first.AnotherValue = 10;
                first.Save();
                TestObject second = new TestObject(Session);
                second.Name = "Second";
                second.Value = 15;
                second.AnotherValue = 3;
                second.Save();
                TestObject third = new TestObject(Session);
                third.Name = "Third";
                third.Value = 9;
                third.AnotherValue = 15;
                third.Save();
            }
        }
    }
}

File: VB/PivotGridCustomization.Module/Updater.vb
Imports Microsoft.VisualBasic
Imports System

Imports DevExpress.ExpressApp.Updating
Imports DevExpress.Xpo
Imports DevExpress.Data.Filtering
Imports DevExpress.ExpressApp
Imports DevExpress.Persistent.BaseImpl

Namespace PivotGridCustomization.Module
	Public Class Updater
		Inherits ModuleUpdater
		Public Sub New(ByVal objectSpace As ObjectSpace, ByVal currentDBVersion As Version)
			MyBase.New(objectSpace, currentDBVersion)
		End Sub
		Public Overrides Sub UpdateDatabaseAfterUpdateSchema()
			MyBase.UpdateDatabaseAfterUpdateSchema()
			Dim testObject As TestObject = ObjectSpace.FindObject(Of TestObject)(CriteriaOperator.Parse("Name == 'First'"))
			If testObject Is Nothing Then
				Dim first As TestObject = ObjectSpace.CreateObject(Of TestObject)()
				first.Name = "First"
				first.Value = 5
				first.AnotherValue = 10
				first.Save()
				Dim second As TestObject = ObjectSpace.CreateObject(Of TestObject)()
				second.Name = "Second"
				second.Value = 15
				second.AnotherValue = 3
				second.Save()
				Dim third As TestObject = ObjectSpace.CreateObject(Of TestObject)()
				third.Name = "Third"
				third.Value = 9
				third.AnotherValue = 15
				third.Save()
			End If
		End Sub
	End Class
End Namespace
